src/transformers/models/unispeech_sat/modeling_unispeech_sat.py

```diff
@@ -55,7 +55,6 @@
 
 # General docstring
 _CONFIG_FOR_DOC = "UniSpeechSatConfig"
-_PROCESSOR_FOR_DOC = "Wav2Vec2Processor"
```

Comment:
Not able to comment at the exact line, but for `UniSpeechSatForPreTraining`, I think you missed to change `Wav2Vec2FeatureExtractor` (you changed this for another model file `UniSpeechForPreTraining`)